sources/packages/backend/apps/queue-consumers/test/helpers/testing-modules/testing-modules-helper.ts

```diff
@@ -44,11 +47,14 @@ export async function createTestingAppModule(): Promise<CreateTestingModuleResul
     },
   );
   const sshClientMock = createMock<Client>();
+  const casServiceMock = createCASServiceMock();
```

Comment:
Was there a benefit to creating a method instead of using the `createMock<CASService>`?